pnpm-lock.yaml

```diff
@@ -1,4 +1,4 @@
-lockfileVersion: '6.1'
+lockfileVersion: '6.0'
```

Comment:
Remember unless we explicitly update dependencies we need to `pnpm install --frozen-lockfile`. The lock file should not be modified otherwise.

And also please when committing files please quickly check the changes before committing. Webstorm has a lovely UI here for this.